src/app/record/components/top-bar-actions/top-bar-actions.component.html

```diff
@@ -1,23 +1,56 @@
-<div class="my-orcid-icons" *ngIf="platform.columns12">
-  <button mat-icon-button class="orc-font-small-print" (click)="printRecord()">
-    <mat-icon fontSet="material-icons-outlined" class="icon"> print </mat-icon>
-    <ng-container i18n="@@record.printableVersion">
-      Printable version
-    </ng-container>
-  </button>
-  <button mat-icon-button class="orc-font-small-print" *ngIf="!isPublicRecord">
-    <mat-icon fontSet="material-icons-outlined" class="icon">
-      qr_code
-    </mat-icon>
-    <ng-container i18n="@@record.getQR">
-      Get a QR code for your ORCID iD
-    </ng-container>
-  </button>
-  <button mat-icon-button class="orc-font-small-print" *ngIf="!isPublicRecord">
-    <mat-icon fontSet="material-icons-outlined" class="icon"> code </mat-icon>
-    <ng-container i18n="@@record.displayId">
-      Display your iD on other sites
-    </ng-container>
-  </button>
+<div class="container">
+  <div
+    class="is-this-you orc-font-body-small"
+    *ngIf="
+      isPublicRecord &&
+      !(
+        userRecord?.userInfo?.REAL_USER_ORCID ===
+        userRecord?.userInfo?.EFFECTIVE_USER_ORCID
```

Comment:
I believe this should only be shown to not login users.... 
 I am asking about this to Dan on slack

Reply:
https://orcid.slack.com/archives/C01UN9KN74L/p1626104614002100